site/en/blog/site-affiliation/index.md

```diff
@@ -167,9 +168,13 @@ at `/.well-known/assetlinks.json` on the respective domains.
     Allow: /.well-known/
     ```
 
+1.  Submit [a
```

Comment:
I think you should end this with "/viewform" without the querystring so it doesn't automatically go into editing mode (if you have permissions to do that, which I guess is just Googlers).